forward/forward_test.go

```diff
@@ -1,44 +0,0 @@
-package forward
```

Comment:
Since we're not setting Opaque anymore, we don't need this test.  I tested that http://i1382.photobucket.com/albums/ah245/PhotobucketMKTG/Gift%20Guide%202015/Product%20Photos/Photos%20With%20Text/Calendars.png~original works (as mentioned in https://github.com/getlantern/lantern/issues/3344).